cmd/wazero/wazero.go

```diff
@@ -231,11 +243,17 @@ func doRun(args []string, stdOut io.Writer, stdErr logging.Writer, exit func(cod
 	} else if needsGo {
 		gojs.MustInstantiate(ctx, rt)
 		err = gojs.Run(ctx, rt, code, conf)
+	} else {
+		_, err = rt.InstantiateModule(ctx, code, conf)
 	}
 
 	if err != nil {
 		if exitErr, ok := err.(*sys.ExitError); ok {
-			exit(int(exitErr.ExitCode()))
+			exitCode := exitErr.ExitCode()
+			if timeout > 0 && exitCode == sys.ExitCodeDeadlineExceeded {
+				fmt.Fprintf(stdErr, "canceled: wasm binary '%s' exceeded timeout (%v)\n", wasmExe, timeout)
```

Comment:
can we just reuse `exitErr.Error()` https://github.com/tetratelabs/wazero/blob/main/sys/error.go#L59-L68 ?

Reply:
done, although the error becomes slightly more cryptic:

> error: module "" closed with context deadline exceeded (timeout 1ms)

I have still added the timeout